modules/tasks/tasks/src/main/java/org/motechproject/tasks/constants/EventDataKeys.java

```diff
@@ -29,4 +29,7 @@ private EventDataKeys() {
     public static final String REPEAT_INTERVAL_TIME = "repeatIntervalInSeconds";
     public static final String JOB_SUBJECT = "jobSubject";
     public static final String TASK_ID = "task_ID";
+    public static final String TASK_ACTIVITY_ID = "task_activity_ID";
```

Comment:
if these are included in every task action event, we should prefix them at least. I am weary of adding parameters to every event because of the issues we already had in DHIS due to retry metadata being added as task params, we will probably have the same issue with these params sooner or later. Is there a way to avoid this?